style.css

```diff
@@ -0,0 +1,74 @@
+h4 {
+  margin-top: 12px;
+  margin-bottom: 0;
+  font-weight: bold;
+  font-size: 16px;
+  color: #172b4d;
+  font-family: Poppins, sans-serif;
+  line-height: 44px;
+}
+
+footer h4 {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 24px;
+  color: #7f8cff;
+  margin-left: 24px;
+}
+
+footer h4:hover {
+  text-decoration: underline;
+  text-decoration-thickness: 2px;
+  text-decoration-color: #23a3fa;
+}
+
+#socials ul {
+  display: flex;
+  align-items: center;
+  padding-left: 0;
+  margin-left: 24px;
+}
+
+.handel {
+  list-style-type: none;
+  height: 19.96px;
+  width: 19.99px;
+  margin-top: 10px;
+  margin-right: 17.01px;
+}
+
+#Main-content {
+  background-image:
+    url("./Images/Ellipse\ 11.svg"),
+    url("./Images/Rectangle\ 4.svg"),
+    url("./Images/Ellipse\ 3.svg"),
+    url("./Images/Ellipse\ 4.svg"),
+    url("./Images/Rectangle\ 5.svg"),
+    url("./Images/Ellipse\ 13.svg"),
+    url("./Images/Rectangle\ 6.svg");
+  background-position:
+    19px 79px,
+    294px 68px,
+    323px 102px,
+    290px 509px,
+    240px 507px,
+    357px 392px,
+    106px 497px;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 600px;
+}
```

Comment:
Good job at adding the background image as requested by the previous reviewer. Kindly modify the section to follow the design template by sticking to font sizes and colours.

Your design:
![image](https://user-images.githubusercontent.com/20554583/198835830-ee02005c-817f-4866-93f6-9d80620201f8.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/198835852-949aaa31-46d2-479d-81d6-d768fd4a6e61.png)

Note that:

- The text of the intro text does not overlap the background image
- The headline text is bolder than you have it
- There are some missing social icons in your design
- The LET'S CONNECT text together with the social icons are vertically aligned with the intro text and headline text.